mlflow/protos/databricks_tracing.proto

```diff
@@ -16,15 +16,15 @@ import "opentelemetry/proto/trace/v1/trace.proto";
 option py_generic_services = true;
 
 service DatabricksTrackingService {
-  rpc createTrace (CreateTrace) returns (CreateTrace.Response) {
+  rpc createTraceInfo (CreateTraceInfo) returns (TraceInfo) {
     option (rpc) = {
       endpoints: [{
         method: "POST",
-        path: "/mlflow/traces/{location}"
+        path: "/mlflow/traces/{location_id}/{trace_info.trace_id}"
```

Comment:
```suggestion
        path: "/mlflow/traces/{location_id}/{trace_info.trace_id}/info"
```

Isn't this the path? Did this path still work when you tested? https://sourcegraph.prod.databricks-corp.com/databricks-eng/universe/-/blob/mlflow/api/proto/trace_service_v4.proto?L127

Reply:
Oh I see you added the `/info` when actually invoking it. Is that preferred or should the full path be here?